internal/commands/vaultsecrets/integrations/create.go

```diff
@@ -179,8 +198,8 @@ func createRun(opts *CreateOpts) error {
 		body := &preview_models.SecretServiceCreateAwsIntegrationBody{
 			Name: opts.IntegrationName,
 			FederatedWorkloadIdentity: &preview_models.Secrets20231128AwsFederatedWorkloadIdentityRequest{
-				Audience: i.Details[AWSKeys[0]],
-				RoleArn:  i.Details[AWSKeys[1]],
+				Audience: ic.Details[AWSKeys[0]],
+				RoleArn:  ic.Details[AWSKeys[1]],
```

Comment:
Curious if there is a plan in the future to also support access keys for AWS / service account keys for GCP as an alternative authentication method.